frontend/src/metabase/actions/containers/ActionCreator/ActionCreator.tsx

```diff
@@ -181,6 +197,13 @@ function ActionCreator({
           />
         </Modal>
       )}
+
+      {route && (
```

Comment:
This condition means that the custom warning modal will be shown only in model details page.
In [other places](https://github.com/metabase/metabase/issues/34114) which trigger create/edit action modal, the modal does not have its own URL. These inconsistencies will be discussed and acted on separately.